cmd/earthly/satellite_cmds.go

```diff
@@ -309,22 +310,22 @@ func (app *earthlyApp) printSatellitesTable(satellites []satelliteWithPipelineIn
 	}
 
 	t := tabwriter.NewWriter(os.Stdout, 1, 2, 2, ' ', 0)
-	headerRow := []string{" ", "NAME", "PLATFORM", "SIZE", "VERSION", "STATE"} // The leading space is for the selection marker, leave it alone
+	headerRow := []string{" ", "NAME", "PLATFORM", "SIZE", "VERSION", "STATE", "LAST USED", "CACHE"} // The leading space is for the selection marker, leave it alone
```

Comment:
`CACHE` is for cache retention right? I wonder if we even need to list it. Most people will be on a single plan and all satellites will have the same entry. I would just keep it under `inspect` only.

Reply:
I'll move it to appear on `-a` only, since it is still useful here in some contexts.